docs/site/pages/plugins/async-node.mdx

```diff
@@ -72,7 +86,7 @@ const player = new Player({
   </core>
   <react>
 
-The `react` version of the AsyncNodePlugin is identical to using the core plugin. Refer to core usage for handler configuration:
+The `react` version of the AsyncNodePluginPlugin is identical to using the core plugin. Refer to core usage for handler configuration:
```

Comment:
I think this should stay `AsyncNodePlugin`

Reply:
done